### PR TITLE
[ENH] Some missing docstrings for bidsschematools

### PR DESCRIPTION
**Pull request by anibalsolon**
Adding some more docstrings for the bidsschematools.

### CODE REVIEW

**Review by bendhouseart — APPROVED** [review]
Looks good, approval and merge should push to rtd's.

**Comment by anibalsolon** [comment]
hey @TheChymera tagging you here as well, if you'd like to take a look!

File: tools/schemacode/bidsschematools/validator.py [hunk 1]
```diff
@@ -514,7 +514,7 @@ def select_schema_dir(
         according to the respective schema version, e.g. "1.7.0".
         If the path starts with the string "{module_path}" it will be expanded relative to the
         module path.
-    schema_version : str or None
+    schema_version : str | None
```
Comment (effigies) [inline tools/schemacode/bidsschematools/validator.py:517]: Above you went in the other direction.

**Review by effigies — COMMENTED** [review]
Mostly good. We should pick a convention for describing types and stick with it, though.

File: tools/schemacode/bidsschematools/validator.py (hunk 1 above)
Comment (anibalsolon) [reply tools/schemacode/bidsschematools/validator.py:517]: Ah missed this one- it should be an `or`. From what I saw from numpydoc that's what's used

**Review by effigies — APPROVED** [review]

**Pull request merged**